### ISSUE TITLE
Google has disable temporarily Grive session start

### ISSUE THREAD

**Issue by ondonadas** [Grive/grive#359]
Hi guys

Today I started to experience that Google is claiming about grive, and the application (Grive) is disabled temporarily.
Do you are aware of what is going on?

![Selección_093](https://user-images.githubusercontent.com/1852881/68481347-3f7f8500-0237-11ea-99ab-4b3e06eb225b.png)

**Comment by SurajViitk** [comment]
Mine was working 2 days before, but not now. Trying to sign in is giving similiar error in browser. Trying to sync using existing sign in raising "Failed to refresh auth token: HTTP 401" error.

**Comment by romanovzky** [comment]
Same happens with me...

**Comment by PieterDepamelaere** [comment]
Likewise here... Hoping Google approves permission soon though

**Comment by hudsantos** [comment]
Hello my friends,

The solution steps for your issue is right here: https://github.com/Grive/grive/blob/master/README.md

**Comment by ghost** [comment]
@hudsantos That doesn't help; grive2 is the new package, but the tool is still called grive, and it gives this exact same error

**Comment by cptX** [comment]
Any news or suggestions?

**Comment by hudsantos** [comment]
> Any news or suggestions?

The solution steps for your issue is right here: https://github.com/Grive/grive/blob/master/README.md

**Comment by cptX** [comment]
@hudsantos doesn't make sense. You are sending us to the page of the old grive (v1). Can you be more specific?

**Comment by hudsantos** [comment]
> @hudsantos doesn't make sense. You are sending us to the page of the old grive (v1). Can you be more specific?

For sure,

Old grive (v1) README.md reads:

> GRIVE NO LONGER MAINTAINED PLEASE USE GRIVE2
> Grive is DEPRECATED. Please refer to the newest fork Grive2.
> Grive2, WHICH IS ACTIVELY MAINTAINED, is here: https://github.com/vitalif/grive2

..and then sends us to grive2: https://github.com/vitalif/grive2

**Comment by cptX** [comment]
@hudsantos  are you trolling us? Of course we know that grive is deprecated and we are all using grive2. The problem is that grive2 recently stopped working. So please stop spamming us with useless redirections! Last working date for me using grive2 was 19.01.2020

**Comment by hudsantos** [comment]
@cptX thanks for your information.

Not trolling anyone, no. Just answering **respectfully** questions that are being asked here. Sorry if you fell personally trolled. That's definitely not the case my friend.

If you want to solve an issue of grive2, what's the point on opening issue on this deprecated project instead? Shouldn't it be opened within the currently maintained project (grive2), which you're facing the issue. Sorry I just didn't get, really, and appreciate if you can explain, **respectfully**. My suggestion is, in fact, redirect to grive2, where we can possibly get the help we want. Cause there at least have someone maintaining.

As I am not maintainer neither here nor there, I will take your suggestion and stop trying to help this thread.

BTW If you don't want to receive messages from this repo anymore, feel free to just unwatch it. This is quite normal in open source projects.

Sorry again.

Sincerely,

**Comment by cptX** [comment]
@hudsantos You are right, that is a mistake to post in this project. I just did it because it had the same problem I'm facing. I'll check and post in the other group. Thanks!
